src/FakeItEasy/ExpressionExtensions.cs

```diff
@@ -12,39 +12,37 @@ internal static class ExpressionExtensions
         /// <summary>
         /// Evaluates an expression, potentially by compiling it into a delegate and invoking the delegate, but faster if possible.
         /// </summary>
-        /// <notes>
-        /// This method evaluates an expression, but tries to do it in a light-weight way that doesn't compile it into a delegate.
-        /// This is often going to be used in the context of finding out 'what object/value is a some (sub-)expression referring to?'
-        /// 
-        /// Key observation is that walking the expression tree and evaluating directly can be much faster for simple expressions like:
-        /// 
-        /// - constant expressions (null, 1, "easy", etc.)
-        /// 
-        /// - local variables referenced from lambda expressions, such as 'fake' in the A.CallTo line:
-        /// 
-        ///     {
-        ///         var fake = A.Fake&lt;Something&gt;();
-        ///         A.CallTo(() => fake.DoStuff());
-        ///     }
-        ///     
-        /// - Expressions that are simple member accesses (field gets, property gets) on an object or class, such as
-        ///     String.Empty
-        ///     A&lt;SomethingA&gt;.Ignored
-        ///     A&lt;SomethingA&gt;._
-        ///     myObj.someProperty
-        ///     
-        /// - Trivial method calls with no arguments (A.ToString(), B.GetType(), Factory.Create())
-        ///     
-        /// </notes>

```

Comment:
StyleCop complained about blank lines in XML doc comments and suggested replacing them with `<para>` tags... instead I cheated and moved them to a normal comment